tranx/dataset2.py

```diff
@@ -116,10 +116,16 @@ def preprocess_conala_dataset(train_file, test_file, grammar_file, src_freq=3, c
     print('Avg action len: %d' % np.average(action_lens), file=sys.stderr)
     print('Actions larger than 100: %d' % len(list(filter(lambda x: x > 100, action_lens))),
           file=sys.stderr)
-
-    pickle.dump(train_examples,
-                open(os.path.join(out_dir, 'train.all_{}.bin'.format(num_mined)), 'wb'))
-    pickle.dump(full_train_examples, open(os.path.join(out_dir, 'train.gold.full.bin'), 'wb'))
+    #dump_size = int(len(train_examples)/n_files)
+    train_split = np.array_split(np.array(train_examples), n_files)
+    for i, train_out in enumerate(train_split):
+        pickle.dump(list(train_out),open(os.path.join(out_dir, f'train.all_{i}.bin'), 'wb'))
+#    for i in range(n_files-1):
```

Comment:
Just remove the commented code completely to make things cleaner. You can check the old version in the VCS when you need it (just checkout a proper commit).